mappings/net/minecraft/datafixer/schema/Schema100.mapping

```diff
@@ -2,7 +2,8 @@ CLASS net/minecraft/class_1222 net/minecraft/datafixer/schema/Schema100
 	METHOD <init> (ILcom/mojang/datafixers/schemas/Schema;)V
 		ARG 1 versionKey
 		ARG 2 parent
-	METHOD method_5195 targetEntity (Lcom/mojang/datafixers/schemas/Schema;Ljava/util/Map;Ljava/lang/String;)V
+	METHOD method_5195 targetEntityItems (Lcom/mojang/datafixers/schemas/Schema;Ljava/util/Map;Ljava/lang/String;)V
+		ARG 2 entityId
 	METHOD method_5196 targetItems (Lcom/mojang/datafixers/schemas/Schema;)Lcom/mojang/datafixers/types/templates/TypeTemplate;
```

Comment:
Hmm, we might consider different naming style for those that return type templates (used within supplier) and those that call schema register. I would keep `targetXxx` for the schema register one and rename these ones possibly? How do you think?

Reply:
Tbh, I followed this specific method for naming all the others. How should the ones returning type templates be renamed?

Reply:
Hmm, you can just keep these names and we take care of these in a later refactor.